### ISSUE TITLE
[Suggestion] "Simply Lost"

### ISSUE THREAD

**Issue by kstrafe** [dogecoin/dogecoin#345]
Excuse me; I have not contributed so I can not tag this "issue". I would like you to do so for me if possible.

I have long been a player of the MMORPG "MapleStory" (Europe Version), and I have come to observe some facts about its economy that may improve Dogecoin. I think what I am about to propose will:
1. Stabilize Dogecoin's value at a certain point.
2. Halt any deflation AND inflation completely.
3. Give miners an incentive to mine.

My proposal is based upon the transaction fee system in MapleStory. The fees are NOT distributed to the miners, but instead they simply vanish. Because of how the transaction  system of Ð works, we can have 50% of the transaction fee vanish, and 50% go to the miners. This allows miners to gain doge based on transaction fees, as well as block mining. The reason we split the fee 50/50 is because in Maplestory, the MS server does all the transaction work. We need miners to do this work in Ð, so we grant them a part of the fee.

Say we have the scheme:

0-99Ð: 1Ð Fee // To avoid tonnes of microtransactions
100Ð+ -> 0.0025% Fee
40kÐ+ -> 1Ð Fee // 40kÐ with 0.0025% = 1Ð, we cap the fee here

Inspired by 
![maplestory-12-atk-pink-adventure-cape-timbaland](https://f.cloud.github.com/assets/4683721/2452795/f3054da0-aed5-11e3-93d1-67e4b6e63bc4.jpg)

The fact that Dogecoin has an increase of 5.2bÐ per year after the 100b cap, is something I really, really like ( I think it's a fantastic idea, not based on a market %, but just a fixed value, great! ). The thing is that we should not rely on lost wallets alone for the amount of Doge to stabilize at a certain point. From MapleStory I know that some people (with billions of meso (multiple characters, 2^31 - 1 max per character)) may be off their account, and then they come back to buy stuff. The fees really make sure inflation doesn't go overboard and that miners keep mining (grinders keep grinding).

I have observed crackers gaining an unlimited meso supply. Luckily, after the patch, the fees have dropped deflated the currency and balanced it to a level where it is equal to the amount of meso gathered from monsters (miners equivalent in dogecoin).

An example:

Transfer 50mÐ -> loss = 1
Currently, that's equivalent to $38,354.00, with a fee of 0.00076729$

Within 100Ð and the value: 1÷(1−(1−FEE_PERCENTAGE÷100)) We will apply the fee, if it's over or under, we apply a fee of 1Ð.

This will be beneficial to poor shibe, yet still allow the rich to transfer billions of Doge with only a 1Ð fee.

Or a fee of choice over the minimum fee.
